draft-ietf-ppm-dap-taskprov.md

```diff
@@ -301,6 +311,34 @@ unimplemented VDAF).
 The definition of `Time`, `Duration`, `Url`, and `BatchMode` follow those in
 {{!DAP}}.
 
+## Extensions {#taskbind-extensions}
+
+The `TaskConfig` structure includes a list of extensions. In general,
+extensions can be used to bind additional, application-specific information to
+the task. For example, an extension might be used to encode the identity of the
+Collector. (Only the Aggregators are identified in `TaskConfig`.)
+
+Each extension is structured as follows:
+
+~~~ tls-presentation
+struct {
+  ExtensionType extension_type;
+  opaque extension_data<0..2^16-1>;
+} Extension;
+
+enum {
+  reserved(0),
+  (2^16-1)
```

Comment:
nit: the doc mentions DAP's ExtensionType which uses 65535, we should make it consistent, probably use 2^16-1 everywhere since that's what the other enum definition uses in this doc.